flink/v1.12/flink/src/main/java/com/netease/arctic/flink/write/ArcticFileWriter.java

```diff
@@ -126,6 +157,11 @@ private Long getTransactionId() {
     return transaction;
```

Comment:
Maybe adding a log message to show how to produce the signature is better. So it is clear what combines the signature.